test/tests/haxe-hello-world/run.sh

```diff
@@ -0,0 +1 @@
+../run-haxe-in-container.sh
```

Comment:
Oh.. I didn't realize it should be a symlink since when I looked at the other run.sh on Windows, they were just files.
I've fixed that, ran `./test.sh` and the tests passed now.